modules/ROOT/partials/module-loading/rollupjs-dev-dependencies.adoc

```diff
@@ -20,19 +20,20 @@ npm install --save-dev \
   rollup-plugin-postcss \
   rollup-plugin-string
 ----
-{% else %}
-1. Add `tinymce` and the following development dependencies to the project.
-
-- [`postcss`](https://www.npmjs.com/package/postcss)
-- [`rollup`](https://www.npmjs.com/package/rollup)
-- [`@rollup/plugin-commonjs`](https://www.npmjs.com/package/@rollup/plugin-commonjs)
-- [`@rollup/plugin-node-resolve`](https://www.npmjs.com/package/@rollup/plugin-node-resolve)
-- [`rollup-plugin-css-bundle`](https://www.npmjs.com/package/rollup-plugin-css-bundle)
-- [`rollup-plugin-postcss`](https://www.npmjs.com/package/rollup-plugin-postcss)
-- [`rollup-plugin-string`](https://www.npmjs.com/package/rollup-plugin-string)
+endif::[]
+ifeval::[{is_zip_install} == false]
```

Comment:
Same issue here as previously, as this isn't the same.
```suggestion
ifeval::[{is_zip_install} != true]
```

Reply:
done